src/internal/operations/test/helper_test.go

```diff
@@ -625,3 +623,25 @@ func newIntegrationTesterSetup(t *testing.T) intgTesterSetup {
 
 	return its
 }
+
+func getTestExtensionFactories() []extensions.CreateItemExtensioner {
+	return []extensions.CreateItemExtensioner{
+		&extensions.MockItemExtensionFactory{},
+	}
+}
+
+func verifyExtensionData(
+	t *testing.T,
+	itemInfo details.ItemInfo,
+	p path.ServiceType,
+) {
+	assert.NotNil(t, itemInfo.Extension, "nil extension")
+	assert.NotNil(t, itemInfo.Extension.Data["NumBytes"], "key not found in extension")
+	eSize := int64(itemInfo.Extension.Data["NumBytes"].(float64))
+
+	if p == path.SharePointService {
+		assert.Equal(t, itemInfo.SharePoint.Size, eSize, "incorrect data in extension")
```

Comment:
For consistency throughout our code, assertions should be written as `assert.Foo(t, expected, actual)`.  This has it backwards.

Reply:
I think the order is correct? `itemInfo.SharePoint.Size` is the expected size.

Reply:
Oh, is it?  I just read `eSize` and thought `expectedSize`.

Reply:
:) that was supposed to be `extensionSize`